web/app/components/comment/comment.jsx

```diff
@@ -264,6 +268,27 @@ export default class Comment extends Component {
     }
   }
 
+  handleVoteError(e, originalVotingState) {
+    let errorMessage = extractErrorMessageFromResponse(e.response);
+    if (errorMessage.indexOf('user dev_user already voted for') === 0) {
```

Comment:
It's strange that you have dev stuff in the code. Is it correct here? Why?

Reply:
Sorry, it was my mistake. I've already fixed it and pushed the corrent version.